clr-installer/main.go

```diff
@@ -482,15 +588,6 @@ func execute(options args.Args) error {
 	// or we get a SIGTERM from reboot
 	signal.Reset()
 
-	if options.Reboot && installReboot {
-		_ = lock.Unlock()
-		if err := cmd.RunAndLog("reboot"); err != nil {
-			if errLog := md.Telemetry.LogRecord("reboot", 1, err.Error()); errLog != nil {
-				log.Error("Failed to log Telemetry fail record: reboot")
-			}
-			return err
-		}
-		log.RequestCrashInfo()
-	}
-	return nil
+	return processRebootOption(options, installReboot, md)
+
```

Comment:
Please drop this extra blank line at the end of the function

Reply:
Thanks done!